File: plugin-sample-java/src/main/java/com/bytex/android/plugin/sample/SampleClassVisitor.java
package com.bytex.android.plugin.sample;

import com.ss.android.ugc.bytex.common.visitor.BaseClassVisitor;

public class SampleClassVisitor extends BaseClassVisitor {

    private SampleExtension extension;

    public SampleClassVisitor(SampleExtension extension) {
        this.extension = extension;
    }

    @Override
    public void visitSource(String source, String debug) {
        super.visitSource(source, debug);
    }
}

File: plugin-sample-kotlin/src/main/java/com/bytex/android/plugin/sample/SampleClassVisitor.kt
package com.bytex.android.plugin.sample

import com.ss.android.ugc.bytex.common.visitor.BaseClassVisitor

class SampleClassVisitor(val extension: SampleExtension): BaseClassVisitor() {

    override fun visitSource(source: String?, debug: String?) {
        super.visitSource(source, debug)
//        println("plugin-sample-kotlin --------> source: $source")
//        println("plugin-sample-kotlin --------> debug: $debug")
    }
}
